pkg/mountutil/mountutilmock/volumestore.mock.go

```diff
@@ -92,12 +92,12 @@ func (m *MockVolumeStoreMockRecorder) List(size any) *gomock.Call {
 }
 
 // Remove mocks the Remove method of VolumeStore
-func (m *MockVolumeStore) Remove(names []string) ([]string, error) {
+func (m *MockVolumeStore) Remove(names []string) ([]string, []error, error) {
 	m.ctrl.T.Helper()
 	ret := m.ctrl.Call(m, "Remove", names)
 	ret0, _ := ret[0].([]string)
 	ret1, _ := ret[1].(error)
-	return ret0, ret1
+	return ret0, []error{}, ret1
```

Comment:
@AkihiroSuda I am not familiar with the mock library, and not sure what I should do here.
I did match the signature with the interface, but should I return anything instead of []error{}?

Reply:
idk, probably just an empty list is fine